examples/long_running_measurement/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "long_running_measurement"
+version = "0.5.0"
+description = "MeasurementLink example that performs a long running measurement."
+authors = ["National Instruments"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+ni-measurementlink-service = {version = "^1.1.0a0", allow-prereleases = true}
+click = ">=7.1.2"
+
+[tool.poetry.group.dev.dependencies]
+mypy = ">=1.0"
+# Uncomment to use prerelease dependencies (requires poetry>=1.2).
+ni-measurementlink-service = {path = "../..", develop = true}
```

Comment:
Is this supposed to be uncommented for now until we actually release 1.1.0a0? Or should it be commented out?